ui/src/app/data-explorer/components/widgets/time-series-chart/config/time-series-chart-widget-config.component.ts

```diff
@@ -146,6 +146,16 @@ export class TimeSeriesChartWidgetConfigComponent extends BaseWidgetConfig<
                 },
             );
         config.showSpike ??= true;
+        config.leftAxis ??= {
```

Comment:
Does this mean that automatic scaling is active and a minimum and a maximum are set? Or am I misunderstanding this? If this is the case, why do both have to be set?

Reply:
Yes, it's just the default value in case autoScale is deselected